tt_metal/llrt/hal.cpp

```diff
@@ -54,6 +54,22 @@ uint32_t Hal::get_programmable_core_type_index(HalProgrammableCoreType programma
     }
 }
 
+std::string_view Hal::get_name(HalProgrammableCoreType core_type) const {
```

Comment:
could remove this and use magic enum utilities

Reply:
somehow I missed that this was already in our code base, will do